Recrutify/Recrutify.DataAccess/Repositories/UserRepository.cs

```diff
@@ -15,6 +17,14 @@ public UserRepository(IOptions<MongoSettings> options)
         {
         }
 
+        public async Task<Dictionary<Guid, string>> GeStaff(IEnumerable<Guid> ids)
+        {
+            var filter = _filterBuilder.In(u => u.Id, ids);
+            var users = await GetCollection().Find(filter).ToListAsync();
```

Comment:
Use Project after Find, to select just necessary fields - id, name, surname

Reply:
done